apps/chat/src/components/Common/ApplicationDialog.tsx

```diff
@@ -432,6 +464,74 @@ const ApplicationDialogView: React.FC<Props> = ({
             )}
           </div>
 
+          <div className="flex flex-col">
+            <label
+              className="mb-1 flex text-xs text-secondary"
+              htmlFor="applicationIcon"
+            >
+              {t('Topics')}
+            </label>
+            <Controller
+              name="topics"
+              control={control}
+              render={({ field: { ref: _ref, ...restField } }) => (
+                <DropdownSelector
+                  {...restField}
+                  placeholder={t('Select one or more topics')}
+                  onChange={handleChangeTopics}
+                  onGetOptionBgColor={(value) =>
+                    getTopicColors(value, {
+                      defaultBgColor: 'var(--bg-layer-4)',
+                    }).backgroundColor
+                  }
+                  onGetOptionBorderColor={(value) =>
+                    getTopicColors(value, {
+                      defaultBorderColor: 'var(--bg-layer-4)',
+                    }).borderColor
+                  }
+                  options={[
+                    { value: 'dev', label: 'dev' },
+                    { value: 'dev1', label: 'dev1' },
+                  ]}
```

Comment:
Do we need this?

Reply:
This is just to represent that dropdown is working